### ISSUE TITLE
New checks : problematic code < ls "${var}/*}" > 

### ISSUE THREAD

**Issue by pchevallereau** [koalaman/shellcheck#1744]
#### For new checks and feature suggestions
- [ x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
path=/tmp
ls "${path}/*"
rm "${path}/*"
```

#### Here's what shellcheck currently says:
$ shellcheck myscript
No issues detected!

#### Here's what I wanted or expected to see:
WARN: bash don't interpret wildcards when you surround it with double quotes

Correct code:
```sh
#!/bin/bash
path=/tmp
ls "${path}"/*
rm "${path}"/*
```
